modules/services/synergy/default.nix

```diff
@@ -0,0 +1,103 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfgC = config.services.synergy.client;
+  cfgS = config.services.synergy.server;
+
+in
+
+{
+  options = {
+
+    services.synergy = {
+      client = {
+        enable = mkOption {
+          default = false;
```

Comment:
Missing a type, same with a bunch of other options. Most of those are probably `types.str`.